pyfftw/interfaces/_utils.py

```diff
@@ -97,24 +101,25 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
         # take the transform of the input array! (in general, we have
         # to assume that the input array will be destroyed during 
         # planning).
-        a_copy = a.copy()
+        if overwrite_input:    # Note: reload_after_transform is not True...
+            args = (a, s, axes) + args
+            a_copy = a.copy()  # ... hence this is not an additional copy
+        else:
+            args = (numpy.empty_like(a), s, axes) + args
```

Comment:
Oh, I see.  You probably mean that the cache would not work, but I tricked the cache into saving the original strides:-)  However, there might be a problem (at least from a performance standpoint) applying a plan done on a strided array to an unstudied array, so I should still try to find a fix.